Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 GIT
   remote: https://github.com/Energy-Sparks/energy-sparks_analytics.git
-  revision: 7e7b3bf3b23f2e8329bc4673af0d2ad1fbd31f8e
-  tag: 5.0.6
+  revision: feb77f32acc38c556f6a1353f12a2b7b693198db
+  branch: monthly-charts-with-up-to-a-year
```

Comment:
Is this supposed to be a branch not a tag?

Reply:
I'll bump this when I've merged.